pkg/rke/templates/templates.go

```diff
@@ -243,9 +244,11 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.25.0-rancher0 <1.25.9-rancher2-1":     aciv5235,
 			">=1.25.9-rancher2-1 <1.25.9-rancher2-2":   aciv5236,
 			">=1.25.9-rancher2-2 <1.25.13-rancher1-1":  aciv5271,
-			">=1.25.13-rancher1-1 <1.26.0-rancher0":    aciv6031,
+			">=1.25.13-rancher1-1 <1.25.16-rancher2-2": aciv6031,
+			">=1.25.16-rancher2-2 <1.26.0-rancher0":    aciv6032,
 			">=1.26.0-rancher0 <1.26.8-rancher1-1":     aciv5271,
-			">=1.26.8-rancher1-1":                      aciv6031,
+			">=1.26.8-rancher1-1 <1.26.11-rancher2-1":  aciv6031,
```

Comment:
Looks like a typo in range here as `1.26.11-rancher2-1` is then "orphaned"
```suggestion
			">=1.26.8-rancher1-1 <1.26.11-rancher2-2":  aciv6031,
```

Reply:
Fixed it.